### PR TITLE
Made readme more culture neutral.

### PR DESCRIPTION
**Pull request by montera82**
#### Description
Made readme more professional.

### CODE REVIEW

**Comment by montera82** [comment]
hi @domodwyer

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ go get -v github.com/domodwyer/mailyak
 mail := mailyak.New("mail.host.com:25", smtp.PlainAuth("", "user", "pass", "mail.host.com"))
 
 mail.To("dom@itsallbroken.com")
-mail.From("nigerianprince@justneedshelp.com")
+mail.From("jsmith@example.com")
```
Comment (topriddy) [inline README.md]: Should you also change the theme of the example from been about 419?

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ go get -v github.com/domodwyer/mailyak
...
 mail.FromName("Prince Fournineteen")
```
Comment (topriddy) [inline README.md]: Needs updating to reflect a new theme.

**Comment by domodwyer** [comment]
Hi @montera82 / @pretizy 

I really didn't give this as much consideration as I should have - please know no offence was meant! 

I'll happily change the texts - do you want me to merge this PR and change tests/examples after, or would you prefer I wait for your changes?

My apologies,

Dom

**Comment by montera82** [comment]
Hi @domodwyer, 
Cool thanks, since i'm at it, i'll add a change to the texts in the examples and ping you to merge 👍

**Comment by montera82** [comment]
Hi @domodwyer, 
I have now cleaned up in the example file, as that is the only other place i have checked so far.. it looks good for a merge now.
Thanks

**Comment by domodwyer** [comment]
Fantastic! 

Thank @montera82, it's really appreciated - and sorry again!

Dom

**Pull request merged**